backend/blueprint/api/migrations/versions/60f668afadb7_.py

```diff
@@ -25,8 +25,11 @@ def upgrade():
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.create_table('User',
-        sa.Column('username', sa.VARCHAR(length=80), autoincrement=False, nullable=False),
-        sa.Column('password', sa.VARCHAR(length=100), autoincrement=False, nullable=True),
-        sa.Column('name', sa.VARCHAR(length=80), autoincrement=False, nullable=True),
-        sa.PrimaryKeyConstraint('username', name='User_pkey'))
+                    sa.Column('username', sa.VARCHAR(length=80),
```

Comment:
No need of this as the previous length was fine according to hound configurations :+1:

Reply:
@ParthS007 I ran `autopep8` over whole code to remove violations

Reply:
@gabru-md Changed as in? I already stated the reason for the difference. May I know what your doubt is?

Reply:
Please use flake 8 and undo the changes :+1: